### DISCUSSION TITLE
Setting to Update Individual Package.jsons instead of the whole Repo

### DISCUSSION THREAD

**Discussion by zackerydev** [renovatebot/renovate#19795] [Request Help] [answered]
👋 I have a pnpm mono repo that I'm using RenovateBot with

Originally when I set it up I was pretty sure I had found a setting that had setup renovatebot to treat each individual package.json as a separate renovate project, so it would raise a PR bumping `lodash` in:
`apps/my-api/package.json` and `apps/my-api2/package.json` as separate PRs.

I can't for the life of me find this setting.

Anyone have ideas/what I'm missing? Or am I completely misremembering?

**Comment by HonkingGoose** [comment, marked as answer]
I found some config options that are related to what you want to do. 😉 

## `additionalBranchPrefix`

I think `additionalBranchPrefix` [^docs3] is the closest match for what you want:

> Here's an example where `additionalBranchPrefix` can help you. Say you're using a monorepo and want to split pull requests based on the location of the package definition, so that individual teams can manage their own Renovate pull requests. This can be done with this configuration:
>
> ```json
> {
>   "additionalBranchPrefix": "{{parentDir}}-"
> }
> ```

If this is not what you want, then keep reading. 😉 

## `includePaths`

Maybe you're thinking of the `includePaths` [^docs] config option?

> If you wish for Renovate to process only select paths in the repository, use `includePaths`.
>
> Alternatively, if you need to just _exclude_ certain paths in the repository then consider `ignorePaths` instead. If you are more interested in including only certain package managers (e.g. `npm`), then consider `enabledManagers` instead.

## Path settings in `packageRules`

Also our `packageRules` [^docs2] config option docs has information about setting paths.

[^docs]: https://docs.renovatebot.com/configuration-options/#includepaths
[^docs2]: https://docs.renovatebot.com/configuration-options/#packagerules
[^docs3]: https://docs.renovatebot.com/configuration-options/#additionalbranchprefix
